src/rust/ensogl/lib/text/src/component/area.rs

```diff
@@ -569,7 +569,7 @@ impl Area {
             eval_ sels_cut (m.buffer.frp.delete_left());
 
             eval_ input.paste (m.paste());
-            eval input.paste_string ((s) m.buffer.frp.paste(m.decode_paste(s)));
+            eval input.paste_string([m](s) m.paste_string(s));
```

Comment:
you don't need `[m]`. Our FRP rules look if your expression starts with `foo.` if it starts, then they use `[foo]` implicitly if not provided explicitly.